R/reporter-summary.R

```diff
@@ -41,22 +43,23 @@ SummaryReporter <- R6::R6Class("SummaryReporter", inherit = Reporter,
     },
 
     add_result = function(context, test, result) {
+      dot <- NULL
```

Comment:
I don't particularly like the flow of logic here, but I can't see an obviously better approach.